### PR TITLE
add monitor for fisco-bcos 3.0

### PR DESCRIPTION
**Pull request by yinghuochongfly**
add monitor feature

### CODE REVIEW

File: tools/BcosAirBuilder/build_chain.sh [hunk 1]
```diff
@@ -348,6 +356,33 @@ download_bin()
...
+download_monitor_bin()
+{
+    if [ ! -z "${mtail_binary_path}" ];then
+        LOG_INFO "Use binary ${mtail_binary_path}"
+        return
+    fi
+    if [ "${x86_64_arch}" != "true" ];then exit_with_clean "We only offer x86_64 precompiled fisco-bcos binary, your OS architecture is not x86_64. Please compile from source."; fi
+    mtail_binary_path="bin/${mtail_binary_name}"
+    package_name="${mtail_binary_name}_${compatibility_mtail_version}_Linux_x86_64.tar.gz"
+    if [ -n "${macOS}" ];then
```
Comment (cyjseagull) [inline tools/BcosAirBuilder/build_chain.sh:368]: We can support mtail for macos when using fisco-bcos air.

File: tools/BcosAirBuilder/build_chain.sh [hunk 2]
```diff
@@ -348,6 +356,33 @@ download_bin()
...
 }
 
+download_monitor_bin()
+{
+    if [ ! -z "${mtail_binary_path}" ];then
+        LOG_INFO "Use binary ${mtail_binary_path}"
+        return
+    fi
+    if [ "${x86_64_arch}" != "true" ];then exit_with_clean "We only offer x86_64 precompiled fisco-bcos binary, your OS architecture is not x86_64. Please compile from source."; fi
+    mtail_binary_path="bin/${mtail_binary_name}"
```
Comment (cyjseagull) [inline tools/BcosAirBuilder/build_chain.sh:366]: Recommend to download mtail to `~/.fisco/mtail`, refer to the tassl download path.

File: tools/BcosAirBuilder/build_chain.sh [hunk 3]
```diff
@@ -429,20 +464,25 @@ Usage:
     -A <Auth mode>                      Default off. If set -A, build chain with auth, and generate admin account.
     -a <Auth account>                   [Optional when Auth mode] Specify the admin account address.
     -w <WASM mode>                      [Optional] Whether to use the wasm virtual machine engine, default is false
+    -m <fisco-bcos monitor>             [Optional] node monitor or not, default is false
+    -i <fisco-bcos monitor ip/port>     [Optional] When expanding the node, should specify ip and prot
```
Comment (cyjseagull) [inline tools/BcosAirBuilder/build_chain.sh:468]:
```suggestion
    -i <fisco-bcos monitor ip/port>     [Optional] When expanding the node, should specify ip and port
```

File: tools/BcosAirBuilder/build_chain.sh [hunk 4]
```diff
@@ -429,20 +464,25 @@ Usage:
...
     -h Help
 
 deploy nodes e.g
     bash $0 -p 30300,20200 -l 127.0.0.1:4 -o nodes -e ./fisco-bcos
+    bash $0 -p 30300,20200 -l 127.0.0.1:4 -o nodes -e ./fisco-bcos -m
     bash $0 -p 30300,20200 -l 127.0.0.1:4 -o nodes -e ./fisco-bcos -s
 expand node e.g
-    bash $0 -C expand -c config -d config/ca -o nodes/127.0.0.1/node5 -e ./fisco-bcos
+    bash $0 -C expand -c config -d config/ca -o nodes/127.0.0.1/node5 -e ./fisco-bcos 
+    bash $0 -C expand -c config -d config/ca -o nodes/127.0.0.1/node5 -e ./fisco-bcos -m -i 127.0.0.1:5 -M monitor/prometheus/prometheus.yml
```
Comment (cyjseagull) [inline tools/BcosAirBuilder/build_chain.sh:478]: If prometheus deploy in another machine, what effect by specify the `prometheus.yml`.

File: tools/BcosAirBuilder/build_chain.sh [hunk 5]
```diff
@@ -624,6 +683,17 @@ if [ -z \${node_pid} ];then
     echo " \${node} isn't running."
     exit 0
 fi
+
+#启动监控程序
```
Comment (cyjseagull) [inline tools/BcosAirBuilder/build_chain.sh:687]:
```suggestion
#Stop monitor here
```

File: tools/BcosAirBuilder/build_chain.sh [hunk 6]
```diff
@@ -589,6 +636,18 @@ export RUST_LOG=bcos_wasm=error
 cd \${SHELL_FOLDER}
 node=\$(basename \${SHELL_FOLDER})
 node_pid=${ps_cmd}
+
+#启动监控程序
```
Comment (cyjseagull) [inline tools/BcosAirBuilder/build_chain.sh:640]:
```suggestion
#start monitor
```

File: tools/BcosAirBuilder/build_chain.sh [hunk 7]
```diff
@@ -641,6 +711,246 @@ EOF
...
+    local pid="pid"
+    local log_cmd="tail -n20  nohup.out"
+    local check_success="\$(${log_cmd} | grep Listening)"
+
+    mkdir -p $(dirname $output/mtail/node.mtail)
+    cat <<EOF >> "${output}/mtail/node.mtail"
+hidden text host
+host = "${ip}"
+
+#节点
```
Comment (cyjseagull) [inline tools/BcosAirBuilder/build_chain.sh:731]:
```suggestion
#node
```

File: tools/BcosAirBuilder/build_chain.sh [hunk 8]
```diff
@@ -641,6 +711,246 @@ EOF
...
+hidden text node
+node = "${node}" 
+
+#链id
```
Comment (cyjseagull) [inline tools/BcosAirBuilder/build_chain.sh:735]:
```suggestion
#chain id
```

File: tools/BcosAirBuilder/build_chain.sh [hunk 9]
```diff
@@ -641,6 +711,246 @@ EOF
...
+hidden text chain
+chain = "chain0"
+
+
+
+#账本id
```
Comment (cyjseagull) [inline tools/BcosAirBuilder/build_chain.sh:741]:
```suggestion
#group id
```

File: tools/BcosAirBuilder/build_chain.sh (hunk 1 above)
Comment (yinghuochongfly) [reply tools/BcosAirBuilder/build_chain.sh:368]: mtail only support linux and windows

**Pull request closed without merging**